hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Alert.java

```diff
@@ -63,6 +63,9 @@
     private List<Note> notes;
 
     @JsonInclude(Include.NON_EMPTY)
+    private List<LifeCycle> lifecycle;
```

Comment:
Before continuing the review I guess I don't understand the motivation for adding LifeCycle to Alert.  Alert already had an ackTime and a resolvedTime, why not use these for the desired filtering?   We had made a conscious decision to keep lifecycle handling very light and I'm not aware of a need to support further lifecycle states.